test/cppsuite/test_harness/workload/workload_tracking.cpp

```diff
@@ -173,4 +189,39 @@ workload_tracking::save_schema_operation(
         testutil_die(EINVAL, error_message.c_str());
     }
 }
+
+int
+workload_tracking::save_operation(const tracking_operation &operation,
+  const uint64_t &collection_id, const char *key, const char *value, wt_timestamp_t ts,
```

Comment:
Please keep c++ types, the key and value should be `std::string` or the typedef `key_value_t` we should probably be using...

Reply:
Fixed in e3b4531df791395ce8775aa9526d396e0c416f54